lib/wallet/rpc.js

```diff
@@ -1568,9 +1568,14 @@ class RPC extends RPCBase {
     const b58 = valid.str(0, '');
     const str = valid.str(1, '');
 
-    const addr = parseHash(b58, this.network);
+    const addr = parseAddress(b58, this.network);
+    if (addr.version !== 0 || addr.hash.length !== 20)
+      throw new RPCError(
+        errs.INVALID_ADDRESS_OR_KEY,
+        'Cannot sign with multisig wallet'
+      );
```

Comment:
- multi-line `if` should be inside `{}'s`
- I think the error message can be improved. The exact problem is more like this: `'Version 0 pubkeyhash address required for signing.'` (and in fact if you wanted to be slick I think you could replace the if condition with `Address.isPubkeyhash()`)

Reply:
Yeah we should. This is the style around the code so far.
Relevant: https://blog.codecentric.de/en/2014/02/curly-braces/